packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -2001,65 +2034,172 @@ export function writeEndSegment(
 //   }
 // }
 //
-// function completeBoundary(suspenseBoundaryID, contentID) {
-//   // Find the fallback's first element.
-//   const suspenseIdNode = document.getElementById(suspenseBoundaryID);
-//   const contentNode = document.getElementById(contentID);
-//   // We'll detach the content node so that regardless of what happens next we don't leave in the tree.
-//   // This might also help by not causing recalcing each time we move a child from here to the target.
-//   contentNode.parentNode.removeChild(contentNode);
-//   if (!suspenseIdNode) {
-//     // The user must have already navigated away from this tree.
-//     // E.g. because the parent was hydrated. That's fine there's nothing to do
-//     // but we have to make sure that we already deleted the container node.
-//     return;
+// const insertStyles = (function() {
```

Comment:
Is this one gone now as well? Should this comment be removed?

Reply:
Or is it the other copy of this that's unminified below that should be removed? Which one is source of truth?

Reply:
the source of truth are these but I hoisted them into the larger src comment. I assumed there would be some churn here and didn't delete them so I could more rapidly develop them. I'll cleanup after we feel the implementation is right